node/src/neighborhood/node_record.rs

```diff
@@ -329,6 +329,7 @@ pub struct NodeRecordMetadata {
     pub desirable_for_exit: bool,
```

Comment:
This also cries as if it was wrong...
And I think it might be because to my understanding this boolean flag was exactly what we used to think should be responsible for the exit service reputation. Now, if we traverse to the numeric representation of the desirability and when we know how we want to deal with unreachable web sites, protecting the certain Node from a strict ban, we suddenly have no use for this flag and leaving the field in there isn't fine. 

I'm convinced it should basically work as replacement of that filed by your new one.